src/style.css

```diff
@@ -1,3 +1,79 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap';
+
 body {
   margin: 0;
+  background-color: #e1e3e1;
+  font-family: Roboto, system-ui;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+  align-items: center;
+}
+
+.img {
```

Comment:
Avoid using class names that describe styles or tag names such as '.img'. Instead, use a class name that describes the purpose or content of the image.